specificatie/gba-openapi.yaml

```diff
@@ -0,0 +1,142 @@
+openapi: 3.0.3
+servers:
+  - description: "APILAB testserver historie"
+    url: https://www.haalcentraal.nl/haalcentraal/api/brphistorie
+info:
+  title: BBRP historie bevragen
+  description: |
+                API voor het zoeken en raadplegen van historische verblijfplaatsen, partners, nationaliteiten en verblijfstitels uit de BRP (inclusief de RNI).
+
+                Zie de [Functionele documentatie](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-historie-bevragen/tree/v1.0.0/features) voor nadere toelichting.
+  version: 2.0.0 (develop)
+  contact:
+    url: https://github.com/VNG-Realisatie/Haal-Centraal-BRP-historie-bevragen
+  license:
+    name: European Union Public License, version 1.2 (EUPL-1.2)
+    url: https://eupl.eu/1.2/nl/
+tags:
+  - name: BRP historie bevragen
+paths:
+  /verblijfplaatshistorie:
+    post:
+      description: |
+        Zoek de verblijfplaatshistorie van een persoon op de opgegeven peildatum of binnen de opgegeven periode.
+        Als je geen peildatum of periode opgeeft, krijg je alle verblijfplaatsen terug. Het meest actuele adres staat bovenaan.
+        
+        Zoek met burgerservicenummer
+      operationId: GetVerblijfplaatshistorie
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/HistorieQuery"
```

Comment:
kan je niet die van openapi.yaml hergebruiken? Dan hoef je hetzelfde niet te herhalen en weten de definitie van requests in gba en proxy versies gelijk zijn

Reply:
Wordt nu hergebruikt uit openapi.yaml. Als gevolg daarvan zijn ook RaadpleegMetPeildatum en RaadpleegMetPeriode verwijderd.